README.rst

```diff
@@ -449,9 +492,17 @@ The following example shows the different configuration options available in
 the run step:
 
 .. code:: yaml
-
+  
+  version: 2.0
+  # Optional map to specify the buildrunner configuration syntax version
   steps:
     my-build-step:
+      depends:
+        - test-step
+        - validation-step
+        # Optional list of steps to specify which steps need to be completed before this step.
+        # The 'version' map must be present and have a value greater than or equal to 2.0.
+
```

Comment:
It's not clear to me from the docs if `depends` will work without the `version: 2.0` in there? What happens when depends is used without the version number?

Reply:
I reworded it as follows, hopefully it is more clear now.

      # Optional step dependency definition to specify which steps need to be completed before this step.
      # The `version` must be present and set to `2.0` or higher for buildrunner to utilize the step dependencies list.
      # An buildrunner error will occur if `depends` is present but `version` is missing or value is lower than `2.0`.